extensions/image-preview/media/main.js

```diff
@@ -153,6 +154,13 @@
 	});
 
 	container.addEventListener('click', (/** @type {MouseEvent} */ e) => {
+		// Activate after first click.
+		if (!isActive) {
+			isActive = true;
+			document.body.classList.add('ready');
```

Comment:
Also consider the case where you have two images side-by-side. You should be able to click back and forth to switch between them without zooming. This logic only handles the very first click

Reply:
Good point, I've now handled state such that you can only zoom once "focused". Since there doesn't seem to be a great way to obtain whether the current pane is focused directly within the webview, first click is emulating focus. I tried a few approaches including `postMessage` from the preview class but settled on emulating focus this way.